auth-foundation/src/main/java/com/okta/authfoundation/client/internal/NetworkUtils.kt

```diff
@@ -117,6 +122,47 @@ internal suspend fun <T> OidcConfiguration.internalPerformRequest(
     }
 }
 
+internal suspend fun OidcConfiguration.internalGetResponseWithRateLimitHandlingFor(
```

Comment:
I think we should rename this (and make it private?) Maybe just `executeRequest`? Open to ideas. But I don't think we need to have the rate limit name in it, that's an implementation detail.

Reply:
That does sound better, and captures exactly what it does. Will change the name to that